Packs/AMP/Integrations/CiscoAMPEventCollector/CiscoAMPEventCollector_description.md

```diff
@@ -0,0 +1,6 @@
+To configure the integration, enter your credentials in the following fields:
+- Client ID
+- API Key
+and the Server URL
+
+Hoe to generate credentials, see [here](https://www.cisco.com/c/en/us/support/docs/security/amp-endpoints/201121-Overview-of-the-Cisco-AMP-for-Endpoints.html)
```

Comment:
What I meant is to list the steps needed to generate the creds, you can add a comment in the end saying that for additional info check official docs (https://www.cisco.com/c/en/us/support/docs/security/amp-endpoints/201121-Overview-of-the-Cisco-AMP-for-Endpoints.html)